src/scss/custom/auth/index.scss

```diff
@@ -1,6 +1,6 @@
 @import '../../index.scss';
```

Comment:
Why importing index.scss?

Reply:
To allow an isolated style eco-system without interfering with any other classes or variables elsewhere on the site, while still importing global style properties. It's just a safety precaution and should for the most part not be necessary. Outside of fringe cases, where variables have conflicting styles, or there's been applied global styles to elements rather than through custom classes.

It's a method I adopted in last iteration to not risking namespace conflicts or unwarranted style changes elsewhere on the site.

In my view, there should be a someone in charge in consolidating custom utility classes and variables - and apply them globally rather than what's currently in place. Easier to maintain, to adopt and overall a cleaner workflow experience.

Reply:
Never heard about that, but okey. 

Yeah, need to do a clean up in the scss files. I will take that on my plate and try to get it sorted out.